api/nodes/views.py

```diff
@@ -1881,7 +1881,7 @@ def get_serializer_class(self):
 
     def get_object(self):
         try:
-            return self.get_node().private_links.get(_id=self.kwargs['link_id'])
+            return self.get_node().private_links.get(_id=self.kwargs['link_id'], is_deleted=False)
```

Comment:
Bug fix for VOL's, info about deleted VOL's were being returned through the API - don't worry, the actual VOL itself doesn't work after being deleted, but we still shouldn't return info about it! ENG-52

Reply:
It appears that, on master, [Anon VOLs are being respected by the API after delete](https://www.flowdock.com/app/cos/archiver/messages/41305). Maybe it doesn't give you extra permissions, but it is trying to anonymize.